src/main/resources/migrations/00068_add_psp_test_account_stage_permission.sql

```diff
@@ -0,0 +1,7 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:psp_test_account_stage_update_permission
+INSERT INTO permissions(id, name, description) VALUES (52, 'psp_test_account_stage:update', 'Update PSP Test Account stage');
```

Comment:
Can we use Max(ID)+1 for id similar to https://github.com/alphagov/pay-adminusers/blob/221a984b930e679ea983a6fe043e7766ce7f268a/src/main/resources/migrations/00065_add_agent_initiated_moto_create_permission.sql#L4 so we can avoid hard coding. I understand we have mix of both but going towards max(id) will make it less error prone